bin/get-video-s3-acls.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env -S python3 -u
+# Bang line above passes '-u' to python, for unbuffered output
+# Permissions required:
+#  psql - access to heroku app's postgres
+#  aws s3 - NZSL IAM access
+#  s3:GetObjectAcl permissions or READ_ACP access to the object
+#  https://docs.aws.amazon.com/cli/latest/reference/s3api/get-object-acl.html
+
+import os
+import sys
+import subprocess
+import argparse
+import re
+from pprint import pprint
+
+parser = argparse.ArgumentParser(
+    description="You must setup: An AWS auth means, eg. AWS_PROFILE env var. "
+    "Postgres access details, eg. DATABASE_URL env var."
+)
+# 'Go' mode, args.do_actions
+parser.add_argument(
+    "--do-actions",
+    action="store_true",
+    default=False,
+    required=False,
+    help="Actually perform Delete objects or change ACLs (DESTRUCTIVE operation)",
+)
+parser.add_argument(
+    "--env",
+    default="uat",
+    required=False,
+    help="Environment to run against, eg 'production, 'uat', etc (default: '%(default)s')",
+)
+parser.add_argument(
+    "--pgcli",
+    default="/usr/bin/psql",
+    required=False,
+    help=f"Postgres client path (default: %(default)s)",
+)
+parser.add_argument(
+    "--awscli",
+    default="/usr/local/bin/aws",
+    required=False,
+    help=f"AWS client path (default: %(default)s)",
+)
+parser.add_argument(
+    "--tmpdir",
+    default="/tmp/nzsl",
+    required=False,
+    help=f"Temp dir path (default: %(default)s)",
+)
+args = parser.parse_args()
+
+# Globals
+CSV_DELIMITER = ","
+DATABASE_URL = os.getenv("DATABASE_URL", "")
+AWSCLI = args.awscli
+PGCLI = args.pgcli
+AWS_S3_BUCKET = f"nzsl-signbank-media-{args.env}"
+
+
+def pg_cli(args_list):
+    if not isinstance(args_list, list):
+        args_list = [args_list]
+    return subprocess.run(
+        [PGCLI, "-c"] + args_list + [f"{DATABASE_URL}"],
+        env=os.environ,
+        capture_output=True,
+        check=True,
+        text=True,
+    )
+
+
+def aws_cli(args_list):
+    if not isinstance(args_list, list):
+        args_list = [args_list]
+    return subprocess.run(
```

Comment:
nit: per my comment above, `aws_cli` isn't actually using this logic

```suggestion
def aws_cli(args_list):
    return subprocess.run(
```

Reply:
and Done.